docs/topics/telemetry.md

```diff
@@ -1,9 +1,10 @@
-# [WIP]
+# Telemetry
 
-## Motivation
+## Custom Script Extensions
 
-The aks-engine team is working towards collecting telemetry from custom script extensions (CSEs) used to configure Kubernetes nodes.
-This data will be used to collect aggregate, non-identifiable data to help us answer gather the following information:
+The aks-engine team has enabled collecting telemetry from custom script extensions (CSEs) used to 
```

Comment:
We should reword this to something like 'The aks-engine team has instrumented custom script extensions (CSEs) so collection of telemetry may be enabled'

This seems more neutral instead of leading this this is enabled and then later saying it is actually disabled by default.

Thoughts?